mywc.c

```diff
@@ -7,13 +7,27 @@
 int main(int argc, char *argv[]) {
     int count = 0;
 
+    printf("Write some text: ");
+
     if (argc == 2 && argv[1][0] == '-' && argv[1][1] == 'c') {
         while (getchar() != EOF) {
             count++;
         }
         printf("\nNumber of characters: %d\n", count);
         return 0;
     }
+    else if (argc == 2 && argv[1][0] == '-' && argv[1][1] == 'w') {
```

Comment:
Please use switch case instead of if requests. So it would be easier to add other statements e.g. -h

Reply:
Added switch-case